osscameroon/camerdevs
# ResponsiveHome: add static pagination design and implementation of th…

…e responsive page

This pull request does...

Why ?
Describe why this pull request is created and what it does

How ?
Let us know how it does what it does

Steps to verify:
Write down few steps to help us test or verify what the pull request does

** Screenshots (optional)
